packages/react-vapor/src/components/select/MultiSelectConnected.tsx

```diff
@@ -183,7 +184,15 @@ class MultiSelect extends React.PureComponent<IMultiSelectProps> {
                 .value();
         }
 
-        return this.props.items.filter((option: IItemBoxProps) => _.contains(this.props.selected, option.value));
+        const selectedItemsWithoutCustom: IItemBoxProps[] = this.props.items.filter((option: IItemBoxProps) =>
+            _.contains(this.props.selected, option.value)
+        );
+        const selectedItemsWithoutCustomItems: string[] = convertItemsBoxToStringList(selectedItemsWithoutCustom);
+        const customItemsValues: string[] = _.difference(this.props.selected, selectedItemsWithoutCustomItems);
+        const customItems: IItemBoxProps[] = convertStringListToItemsBox(customItemsValues);
+        const selectedItems: IItemBoxProps[] = selectedItemsWithoutCustom.concat(customItems);
+
+        return selectedItems;
     }

```

Comment:
Can we get a test for `getSelectedOptions`?

Reply:
Good point, I added two tests to evaluate that if a custom item is stored in the selected, would it be displayed as a SelectedOption.